core/templates/domain/exploration/InteractionObjectFactory.ts

```diff
@@ -121,10 +234,41 @@ export class Interaction {
 })
 export class InteractionObjectFactory {
   constructor(
-    private answerGroupFactory: AnswerGroupObjectFactory,
-    private hintFactory: HintObjectFactory,
-    private solutionFactory: SolutionObjectFactory,
-    private outcomeFactory: OutcomeObjectFactory) {}
+      private answerGroupFactory: AnswerGroupObjectFactory,
+      private hintFactory: HintObjectFactory,
+      private solutionFactory: SolutionObjectFactory,
+      private outcomeFactory: OutcomeObjectFactory,
+      private subtitledHtmlObjectFactory: SubtitledHtmlObjectFactory,
+      private subtitledUnicodeObjectFactory: SubtitledUnicodeObjectFactory
+  ) {}
+
+  convertCustomizationArgsBackendDict(
+      interactionId: string,
+      caValuesBackendDict: IInteractionCustomizationArgsBackendDict
+  ) : IInteractionCustomizationArgs {
+    if (!caValuesBackendDict) {
+      return {};
+    }
+
+    let caValues = angular.copy(caValuesBackendDict);
+
+    let convertToSubtitled:
+      InteractionCustArgsConversionFn<SubtitledBackendDicts, Subtitled> = (
+          caValue, schemaObjType
+      ) => {
+        if (schemaObjType === 'SubtitledHtml') {
+          return this.subtitledHtmlObjectFactory.createFromBackendDict(
+            <ISubtitledHtmlBackendDict> caValue);
+        } else if (schemaObjType === 'SubtitledUnicode') {
+          return this.subtitledUnicodeObjectFactory.createFromBackendDict(
+            <ISubtitledUnicodeBackendDict> caValue);
+        }
+      };
```

Comment:
For instance if you try you do this
```typescript
let x: ISubtitledHtmlBackendDict;
let z = convertToSubtitled(x, 'SubtitledUnicode');
```

There is no error, but should be. You can refer [this file](https://github.com/oppia/oppia/blob/develop/core/templates/domain/statistics/LearnerActionObjectFactory.ts) to achieve that.